src/plugins/new-ui/public/webcomponents/compass.html

```diff
@@ -48,143 +84,265 @@
+            </g>
           </g>
 
           <text id="lock" fill$="{{ options.bezel }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="120px" letter-spacing="0" line-height="100%" word-spacing="0" y="280" x="460.67117" title="Heading hold target">{{__('LK')}}</text>
 
           <text id="heading" fill$="{{ options.headingFill }}" stroke$="{{ options.bezel }}" stroke-width$="{{ options.headingStrokeWidth }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="250px" letter-spacing="0"
           line-height="125%" word-spacing="0" y="530" x="460.67117" font-weight="bold">{{heading}}</text>
-
+          
           <text id="target" fill$="{{ options.bezel }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="150px" letter-spacing="0" line-height="100%" word-spacing="0" y="680" x="460.67117" font-weight="bold"
           title="Heading hold target">{{target}}</text>
 
+          <text id="mode" fill$="{{ options.modeFill }}" stroke$="{{ options.bezel }}" stroke-width$="{{ options.headingStrokeWidth }}" style="letter-spacing:0;text-anchor:middle;word-spacing:0;text-align:center;" font-family="Helvetica Neue,Helvetica,Arial,sans-serif" font-size="200px" letter-spacing="0"
+          line-height="125%" word-spacing="0" y="1206" x="460.67117" font-weight="bold" tabindex="0">{{__(mode)}}</text>
+          
           <g id="indicator">
-            <path fill$="{{ options.indicator }}" d="m 471.41747,-25.529456 c 35.08805,139.334496 35.04692,139.182386 35.04692,139.182386 l -69.01546,0 z" style="fill-opacity:1;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
-            />
+            <path fill$="{{ options.indicator }}" d="m 471.41747,-25.529456 c 35.08805,139.334496 35.04692,139.182386 35.04692,139.182386 l -69.01546,0 z" style="fill-opacity:1;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"/>
           </g>
           <g id="bearing">
             <path fill$="{{ options.bezel }}" style="fill-opacity:1;stroke:#000000;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" d="m 472.79747,253.34433 c -35.08391,-139.0359 -35.0428,-138.88412 -35.0428,-138.88412 l 69.00736,0 z" />
           </g>
         </g>
       </svg>
     </div>
+
+    
   </template>
+  
   <script>
-    var lastData = null;
-    var newData = null;
-    Polymer({
-      is: 'orov-newui-compass',
-      properties: {
-        bearing: {
-          observer: 'setBearing',
-          notify: true
+      var lastData = null;
+      var newData = null;
+
+      //The operating mode of the display
+      var isCompass = true;
```

Comment:
I'd go ahead and make this a first class property. In that sense, the user could default the mode by setting this attribute in the HTML.